### PR TITLE
feat: add securitydefinitions.bearerauth for bearer token setting in V2

### PR DESCRIPTION
**Pull request by XiXiangFiles**
**Describe the PR**
add `securitydefinitions.bearerauth` for [bearer-authentication documentation](https://swagger.io/docs/specification/authentication/bearer-authentication/)

**Relation issue**
https://github.com/swaggo/swag/issues/1853

### CODE REVIEW

**Comment by dwivedisshyam** [comment]
I need this

**Comment by halfstring** [comment]
I need this~

**Comment by GCrean** [comment]
I need this..

**Comment by rjsadow** [comment]
I would love to see this merged.

File: parserv3.go [hunk 1]
```diff
@@ -351,6 +351,13 @@ func parseSecAttributesV3(context string, lines []string, index *int) (string, *
 		search = []string{authorizationURL, in}
 	case secAccessCodeAttr:
 		search = []string{tokenURL, authorizationURL, in}
+	case secBearerAuthAttr:
+		scheme := spec.SecurityScheme{
+			Type:         "http",
+			Scheme:       "bearer",
+			BearerFormat: "JWT",
```
Comment (ubogdan) [inline parserv3.go:358]: We allow only jwt bearers ?

**Review by ubogdan — APPROVED** [review]

File: parserv3.go (hunk 1 above)
Comment (XiXiangFiles) [reply parserv3.go:358]: According to the OpenAPI 3.0 specification, Bearer Token must be used as defined.

If the user wants to use another security scheme, they need to append it under the security section in the YAML configuration.
 image

**Comment by ubogdan** [comment]
@XiXiangFiles Thanks for your contribution !

**Comment by daleiyinsi** [comment]
I nedd this

**Pull request merged**